docs/core/tracer.md

```diff
@@ -49,13 +50,13 @@ The `Tracer` utility must always be instantiated outside of the Lambda handler.
 
 ### Utility settings
 
-The library has one optional setting. You can set it as environment variable, or pass it in the constructor.
-
-This setting will be used across all traces emitted:
+The library has three optional settings. You can set them as environment variables, or pass them in the constructor:
 
 Setting | Description                                                                                    | Environment variable | Constructor parameter
 ------------------------------------------------- |------------------------------------------------------------------------------------------------| ------------------------------------------------- | -------------------------------------------------
+**Tracing enabled** | Enables or disables tracing. By default tracing is enabled when running in AWS Lambda. | `POWERTOOLS_TRACE_ENABLED`          | `enabled`
 **Service name** | Sets an annotation with the **name of the service** across all traces e.g. `serverlessAirline` | `POWERTOOLS_SERVICE_NAME` | `serviceName`
+**Capture HTTPs Requests** | Defines whether HTTPs requests will be traced or not, enabled by default when tracing is also enabled. | `POWERTOOLS_TRACER_CAPTURE_HTTPS_REQUESTS` | `captureHTTPsRequests`
```

Comment:
Outside the scope of this PR: it would be good to document more what are the supported values, the default values and an example of value for each of these.
I created this issue:
https://github.com/awslabs/aws-lambda-powertools-typescript/issues/724

Reply:
Thank you for creating the issue, I like also that you've marked it as `good-first-issue`.